.github/scripts/generate_binary_build_matrix.py

```diff
@@ -45,6 +39,16 @@
     "cpu": "pytorch/manylinux-cpu"
 }
 
+CONDA_CONTAINER_IMAGES = {
```

Comment:
Please consider type-annotating your code
```suggestion
CONDA_CONTAINER_IMAGES : Dict[str, str]= {
```